docs/manual-EN/import_data/nebula-importer-updated.md

```diff
@@ -165,16 +166,16 @@ go run importer.go --config /path/to/yaml/config/file
 
 With `docker`, You can import your local data to **Nebula Graph** with the following command:
 
-```
+```bash
     docker run --rm -ti \
     --network=host \
     -v {your-config-file}:/root/{your-config-file} \
     -v {your-csv-data-dir}:/root/{your-csv-data-dir} \
     vesoft/nebula-importer
     --config /root/{your-config-file}
 ```
-**Note**: You have to change your direcotry to the directory where your `.yaml`configuration is stored. For example, if your `config.yaml` file is in the /home/nebula/ directory, it means {your-config-file} = /home/nebula/config.yaml; if your `csv` file is in the /home/nebula/ directory, it means {your-csv-data-dir} = /home/nebula/.
 
+**Note**: You have to change your direcotry to the directory where your `.yaml`configuration is stored. For example, if your `config.yaml` file is in the /home/nebula/ directory, it means {your-config-file} = /home/nebula/config.yaml; if your `csv` file is in the /home/nebula/ directory, it means {your-csv-data-dir} = /home/nebula/.
```

Comment:
You have to change your directory to where your `.yaml`configuration is stored.
{your-config-file} > {your-config-file-dir}

Reply:
This part is a little tricky, and I would keep the detailed part